addons/notes/src/index.js

```diff
@@ -2,7 +2,7 @@ import deprecate from 'util-deprecate';
 import addons from '@storybook/addons';
 import { WithNotes as ReactWithNotes } from './react';
 
-export const withNotes = ({ notes }) => {
+export const addonNotes = ({ notes }) => {
```

Comment:
It was already a breaking change from the `WithNotes` Component so just a wording change.

I made it because in knobs I couldn't rename the function already name withKnobs and I wanted the same naming convention between addons :)